model/command.go

```diff
@@ -80,7 +81,13 @@ func (o *Command) IsValid() *AppError {
 		return NewAppError("Command.IsValid", "model.command.is_valid.update_at.app_error", nil, "", http.StatusBadRequest)
 	}
 
-	if !IsValidId(o.CreatorId) {
+	// If the CreatorId is blank, this should be a command created by a plugin.
+	if o.CreatorId == "" && !IsValidPluginId(o.PluginId) {
```

Comment:
Having the CreatorId blank might break the frontend when it displays which user the slash command is supposed to be created by. If it's a quick fix, can we just fix that? Or we can make a follow up ticket.

Reply:
Good point. I assume you're talking about the `integrations/commands/installed` list of slash commands? I edited my db, removed the creatorId, and it's fine -- it sends in an empty object if it can't find the creator's user profile: https://github.com/mattermost/mattermost-webapp/blob/1d6ad140c501f21ad5d2c9046845beefdd7fecd3/components/integrations/installed_commands/installed_commands.jsx#L96

If there's any other spots you can think of, I'll test those too.